### ISSUE TITLE
Event date is incorrect in Month View with set timezone

### ISSUE THREAD

**Issue by tsvetomir** [telerik/kendo-angular#1964]
## I'm submitting a...
* Bug report 

## Current behavior
An event starting at 2018-10-21T13:00:00+00:00 would be displayed on Oct 20 in Month View when the Scheduler is set to America/New_York.

## Expected behavior
The event should be displayed on Oct 21 as the [local date](https://www.timeanddate.com/worldclock/converter.html?iso=20181021T130000&p1=1440&p2=179) is 2018-10-21T09:00:00

## Minimal reproduction of the problem with instructions
1. Set your system time zone to America/New_York
1. Open https://stackblitz.com/edit/angular-wnnwas
1. Observe that the event is displayed on Oct 20 in Month View
  ![image](https://user-images.githubusercontent.com/247917/49378300-e5b43800-f714-11e8-8d26-66c6a520c43f.png)
1. Switch to Day View, the event is correctly displayed on Oct 21
  ![image](https://user-images.githubusercontent.com/247917/49378331-fa90cb80-f714-11e8-9cd2-3e49d291dd33.png)

## Environment

Package versions:

```
├── @progress/kendo-angular-scheduler@0.3.2
├── @progress/kendo-date-math@1.3.2
```

Browser:

- Firefox version 63.0.3

**Comment by ipeshev** [comment]
Fix available in [latest development version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/).

**Comment by danielkaradachki** [comment]
Fixed in 0.3.4:
https://stackblitz.com/edit/angular-shekwv?file=app/app.component.ts
